package.json

```diff
@@ -13,7 +13,7 @@
     "lint": "eslint --ext .js,.vue src test/unit/specs"
   },
   "dependencies": {
-    "ol": "^5.3.3",
+    "ol": "^6.1.1",
```

Comment:
Is this required?

Reply:
@Gnafu I don't remember honestly, probably something didn't work.